.changeset/slimy-icons-grin.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/components": patch
+---
+
+Rename directories to match convention (clean up only).
```

Comment:
Do we need to release?

Reply:
I was thinking it's the equivalent to any other refactor, since I renamed some directories. Just in case something breaks.
I can get rid of it though. Not too fussed.

Reply:
Hm doesn't hurt I guess